frontend/src/Routes/Routes.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { createBrowserRouter } from "react-router-dom";
+import App from "../App";
+import TestPage from "../pages/TestPage";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <App />,
+    children: [{ path: "testPage", element: <TestPage /> }],
```

Comment:
Looks correct but I'm trying to test the path /testPage and its not working. What is the correct link to access this?